changelog/db.changelog-1.2.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="John Watson" id="1">
+    <tagDatabase tag="v1.2.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <addColumn tableName="organisations">
+      <column name="published" type="TIMESTAMP"></column>
+    </addColumn>
+    <addColumn tableName="calculations">
+      <column name="published" type="TIMESTAMP"></column>
+    </addColumn>
+  </changeSet>
+  <changeSet author="John Watson" id="3">
+    <dropForeignKeyConstraint baseColumnNames="sbi" baseTableName="calculations" constraintName="fk_organisations_sbi" referencedColumnNames="sbi" referencedTableName="organisations" />
```

Comment:
This is too restrictive for DWH update which may occur in any order.  Eg Calculation may come before Organisation.